cms/static/sass/assets/_graphics.scss

```diff
@@ -13,40 +13,35 @@
 
 .large-advanced-icon {
 	display: inline-block;
-	width: 100px;
-	height: 60px;
-	margin-right: ($baseline/4);
+	width: ($baseline*5-1);
+	height: ($baseline*3);
 	background: url(../images/large-advanced-icon.png) center no-repeat;
 }
 
 .large-discussion-icon {
 	display: inline-block;
-	width: 100px;
-	height: 60px;
-	margin-right: ($baseline/4);
+	width: ($baseline*5-1);
+	height: ($baseline*3);
 	background: url(../images/large-discussion-icon.png) center no-repeat;
 }
 
 .large-html-icon {
 	display: inline-block;
-	width: 100px;
-	height: 60px;
-	margin-right: ($baseline/4);
+	width: ($baseline*5-1);
+	height: ($baseline*3);
 	background: url(../images/large-html-icon.png) center no-repeat;
 }
 
 .large-problem-icon {
 	display: inline-block;
-	width: 100px;
-	height: 60px;
-	margin-right: ($baseline/4);
+	width: ($baseline*5-1);
+	height: ($baseline*3);
 	background: url(../images/large-problem-icon.png) center no-repeat;
 }
 
 .large-video-icon {
 	display: inline-block;
-	width: 100px;
-	height: 60px;
-	margin-right: ($baseline/4);
+	width: ($baseline*5-1);
```

Comment:
Was a fix so that the icon was actually visually contained within the button, instead of popping out one pixel to the right.  In chrome, it made the focus styling have a non-rectangular shape. (In other browsers, this didn't make a difference. Either way, the focus styling was rectangular.)  Maybe there is a more standard solution to this?